Sources/SwiftProtobuf/HashVisitor.swift

```diff
@@ -37,7 +37,9 @@ final class HashVisitor: Visitor {
   init() {}
 
   func visitUnknown(bytes: Data) {
-    mix(bytes.hashValue)
+    if bytes.count > 0 { // Workaround for Linux Foundation bug
```

Comment:
Does it make sense to put a linux test around this as a flag for when it could come out?